openmc/data/grid.py

```diff
@@ -2,7 +2,7 @@
 
 
 def linearize(x, f, tolerance=0.001):
-    """Return a tabulated representation of a function of one variable.
+    """Returns a tabulated representation of a one-variable function
```

Comment:
```suggestion
    """Return a tabulated representation of a one-variable function
```

We generally try to follow [PEP 257](https://www.python.org/dev/peps/pep-0257/#one-line-docstrings) which prefers `Return` (commands) over `Returns` (descriptions).

Reply:
Updated. I can see how `Return` is functionally descriptive.
******
<p align="left">
    <a href="https://www.pullrequest.com/">
        <img width="15%" src="https://static.pullrequest.com/images/dashboard/pullrequest-logo.svg">
    </a>
</p>